sdks/python/sample/conftest.py

```diff
@@ -27,20 +27,24 @@ def test_server_mode(request):
     """
```

Comment:
This is a demonstration of how the test-server-sdk can be used with pytest, things in here can be configured differently when we integrate the test-server-sdk  to python test infrastructure.